### PR TITLE
fix(@schematics/angular): fixes issue that `viewEncapsulation` is not…

### LINKED ISSUE #13689: ng new  --viewEncapsulation Native|None|ShadowDom do nothing
# 🐞 Bug report

### Command (mark with an `x`)
```
- [x] new
```

### Description

Hi,
I try to start new project with default `viewEncapsulation` set to `ShadowDom`.
According to `ng new` documentation, i use `--viewEncapsulation` flag.

But once the project is generated and served, `app-root` does not have `#shadow-root`.
I tried all the other encapsulation but I do not see any result.

**What is the expected result for this flag?**
Set default shematic with `"viewEncapsulation": "ShadowDom"` in `angular.json` and generate `app.component.ts` with `encapsulation: ViewEncapsulation.ShadowDom`
**OR** 
Add to `main.ts` :
```typescript
bootstrapModule(AppModule, {
  defaultEncapsulation: ViewEncapsulation.ShadowDom
})
```
Currently it does neither of the two.

## 🔬 Minimal Reproduction
```bash
ng new view-test --skip-git --skip-install --viewEncapsulation ShadowDom
or
ng new view-test --skipGit --skipInstall --view-encapsulation Native
or
ng new view-test --skipGit --skipInstall --viewEncapsulation None
```

## 🌍 Your Environment
```
@angular/cli@7.3.1

Windows 10 64bit
```

### PR DESCRIPTION
**Pull request by alan-agius4**
… set in `app.component.ts` and in `angular.json`

Fixes #13689

### CODE REVIEW

**Comment by kekel87** [comment]
I think it should also add the option in the `angular.json` file :
```json
"schematics": {
  "@schematics/angular:component": {
    "viewEncapsulation": "ShadowDom"
  }
},
```
For other generated components.

**Comment by alan-agius4** [comment]
@kekel87, yeah make sense.

**Comment by alan-agius4** [comment]
@clydin, or should we set `defaultEncapsulation` inside the `main.ts`?

**Comment by clydin** [comment]
At the application level, I think it would make more sense as a bootstrap option

File: packages/schematics/angular/application/files/src/main.ts.template [hunk 1]
```diff
@@ -7,6 +7,12 @@ import { environment } from './environments/environment';
 if (environment.production) {
   enableProdMode();
 }
-
+<% if(!!viewEncapsulation) { %>
+platformBrowserDynamic().bootstrapModule(AppModule, {
+  defaultEncapsulation: ViewEncapsulation.ShadowDom
```
Comment (clydin) [inline packages/schematics/angular/application/files/src/main.ts.template]: Shouldn't this be the view encapsulation value?

Comment (alan-agius4) [reply packages/schematics/angular/application/files/src/main.ts.template]: done

**Pull request merged**